### PR TITLE
Feature/ add repair allocation tests

### PR DESCRIPTION
**Pull request by Naman-1234**
This pull request covers the issue https://github.com/0chain/system_test/issues/117.

### CODE REVIEW

File: tests/cli_tests/zboxcli_repair_file_test.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+		output, err = uploadFile(t, configPath, map[string]interface{}{
+			"allocation": allocationID,
+			"remotepath": "/",
+			"localpath":  filename,
+		}, true)
+		require.Nil(t, err, strings.Join(output, "\n"))
+		require.Len(t, output, 2)
+
+		require.Nil(t, err, strings.Join(output, "\n"))
+		require.Len(t, output, 2)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_repair_file_test.go:56]: This repeated

File: tests/cli_tests/zboxcli_repair_file_test.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+		// now delete the file from any random blobber
+		allocation := getAllocation(t, allocationID)
+		require.Len(t, allocation.Blobbers, 2)
+
+		// deleting the file in single blobber
+		params := createParams(map[string]interface{}{
+			"allocation": allocationID,
+			"remotepath": "/",
+		})
+		output, err = deleteFile(t, walletOwner, params, false)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_repair_file_test.go:68]: Calling delete will delete it from both blobbers.

File: tests/cli_tests/zboxcli_repair_file_test.go [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+		// Since there were 2 blobbers before, Now they should be 2
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_repair_file_test.go:69]: This comment does not seem related to the following code

File: tests/cli_tests/zboxcli_repair_file_test.go [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+		require.Nil(t, err, strings.Join(output, "\n"))
+		require.Len(t, output, 1)
+		require.Equal(t, fmt.Sprintf("%s deleted", filepath.Base(filename)), output[1])
+
+		// now we will try to repair the file and will create another folder to keep the same
+		err = os.MkdirAll("tmp_repair", os.ModePerm)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_repair_file_test.go:75]: Use os.Temp() for consistency

File: tests/cli_tests/zboxcli_repair_file_test.go [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+		// deleting the file in single blobber
+		params := createParams(map[string]interface{}{
+			"allocation": allocationID,
+			"remotepath": "/",
+		})
+		output, err = deleteFile(t, walletOwner, params, false)
+		// Since there were 2 blobbers before, Now they should be 2
+		require.Nil(t, err, strings.Join(output, "\n"))
+		require.Len(t, output, 1)
+		require.Equal(t, fmt.Sprintf("%s deleted", filepath.Base(filename)), output[1])
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_repair_file_test.go:72]: It should be output[0]

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: tests/cli_tests/zboxcli_repair_file_test.go (hunk 1 above)
Comment (Naman-1234) [reply tests/cli_tests/zboxcli_repair_file_test.go:56]: Removed.

File: tests/cli_tests/zboxcli_repair_file_test.go (hunk 2 above)
Comment (Naman-1234) [reply tests/cli_tests/zboxcli_repair_file_test.go:68]: looking into this

File: tests/cli_tests/zboxcli_repair_file_test.go (hunk 3 above)
Comment (Naman-1234) [reply tests/cli_tests/zboxcli_repair_file_test.go:69]: Removed it.

File: tests/cli_tests/zboxcli_repair_file_test.go (hunk 4 above)
Comment (Naman-1234) [reply tests/cli_tests/zboxcli_repair_file_test.go:75]: Replaced it.

**Comment by Naman-1234** [comment]
Closing this pull request and raised a new one #440

**Pull request closed without merging**